web/unelte/vagrant/webserver/provision/nginx.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -euox pipefail
+
+echo "Dezactivăm firewall-ul."
```

Comment:
E putin inselatoare informatia, `sudo ufw disable` ar fi dezactivarea, aici doar deschizi in port.